llvm/test/Verifier/AMDGPU/intrinsic-amdgpu-init-exec-from-input.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: sed -e '/^; MARK$/,$d' %s | not llc -mtriple=amdgcn -mcpu=gfx942 2>&1 | FileCheck %s
+; RUN: sed -e '1,/^; MARK$/d' %s | llc -mtriple=amdgcn -mcpu=gfx942 -filetype=null
```

Comment:
Do not use llc in verifier tests, only llvm-as.

Also shouldn't need to use sed, simpler to test valid cases in an assembler or codegen test